### DISCUSSION TITLE
is it possible to access Navigator from Brython ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2235] [Q&A] [answered]
Is there a way to access the [WEB API Navigator](https://developer.mozilla.org/en-US/docs/Web/API/Navigator) from Bryhton ?

I've tried few things but none worked :)

Thanks.

**Comment by PierreQuentel** [comment, marked as answer]
Hi,

Yes it's possible, through `window.navigator`:

```python
from browser import window

print(window.navigator.languages)
```

**Reply by BobSquarePants** [reply]
I've tried before with 

`window.navigator.clipboard` but I was getting an Internet Browser console error, but the error was not at all explicit enough..

but now, I know why it was not working
> This feature is available only in [secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) (HTTPS), in some or all [supporting browsers](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard#browser_compatibility).

And as I was testing in local over http:// it could not work..

Thanks @PierreQuentel
